src/opening-period/form/OpeningPeriodForm.tsx

```diff
@@ -354,17 +354,19 @@ export default function OpeningPeriodForm({
           </p>
         </div>
         <div className="opening-period-final-action-row-container">
-          <PrimaryButton
-            dataTest="publish-opening-period-button"
-            className="opening-period-final-action-button"
-            type="submit">
-            Julkaise
-          </PrimaryButton>
-          <SecondaryButton
-            className="opening-period-final-action-button"
-            onClick={(): void => returnToResourcePage()}>
-            Peruuta ja palaa
-          </SecondaryButton>
+          <div className="main-container">
```

Comment:
I think this div could be a exported component in the Main.tsx so we will not accidentally delete or modify it.

Reply:
I think it is pretty much ready then

Reply:
- [ ] Create and apply MainContainer component